themes/socialblue/templates/group/group--statistic--sky.html.twig

```diff
@@ -0,0 +1,40 @@
+<div class="card__counter">
+  <ul>
+    <li>
+      <span class="card__counter-quantity">{{ group_events }}</span>
+      <span class="card__counter-text">{% trans %}event{% plural group_events %}events{% endtrans %}</span>
+    </li>
+    <li>
+      <span class="card__counter-quantity">{{ group_topics }}</span>
+      <span class="card__counter-text">{% trans %}topic{% plural group_topics %}topics{% endtrans %}</span>
+    </li>
+    <li>
+      <span class="card__counter-quantity">{{ group_members }}</span>
+      <span class="card__counter-text">{% trans %}member{% plural group_members %}members{% endtrans %}</span>
+    </li>
+  </ul>
+</div>
+{% if group_operations_url %}
+  <div class="hero-footer__cta">
+    <div class="btn-group">
+      {% if joined %}
+        <button type="button" autocomplete="off" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn btn-accent btn-lg btn-raised dropdown-toggle">{% trans with {'context': 'Is a member'} %}Joined{% endtrans %}<span class="caret"></span></button>
+        <ul class="dropdown-menu dropdown-menu-right">
+          <li><a href="{{ group_operations_url }}">{% trans %}Leave group{% endtrans %}</a></li>
+        </ul>
+      {% elseif closed_group %}
+        <a href="{{ group_operations_url }}" class="btn btn-accent disabled" title="{{ cta }}">{{ cta }}</a>
+      {% else %}
+        <a href="{{ group_operations_url }}" class="btn btn-accent" title="{% trans %}Join{% endtrans %}">{% trans %}Join{% endtrans %}</a>
+      {% endif %}
+    </div>
+  </div>
+{% endif %}
```

Comment:
This feels like a lot of logic that probably shouldn't be in a template file but simply be created with a render array.